### DISCUSSION TITLE
Resizing Calendar Events - seeding event height + resizing events + start time of event

### DISCUSSION THREAD

**Discussion by mrm2234** [tailwindlabs/tailwindcss#11442] [Help]
Hi Tailwind Community, 

I am trying to make events on the calendar resizable - right now, I am copying the code found for the "Week View" [here](https://tailwindui.com/components/application-ui/data-display/calendars). I added "resize-y" to the `a` tag in the `li` component. That works great and the event is resizable but I am unable to resize it down from it's original size. For example, if the event is initially for one hour, I cannot resize it to be 30 minutes. 

My understanding (could be incorrect) is that the height comes from the `gridRow` styling but I am having a hard time figuring out how I can write the styling so that I can reduce the size of the event to a certain minimum. If anyone knows how I could achieve this that would be awesome!

Thanks in advance. 

```
 <ol className=" col-start-1 col-end-2 row-start-1 grid grid-cols-1 sm:grid-cols-7 sm:pr-8"
       style={{ gridTemplateRows: '1.75rem repeat(288, minmax(0, 1fr)) auto' }} >
    <li className="relative mt-px flex sm:col-start-3" style={{ gridRow: '74 / span 18' }} > // issue here
        <a href="#"
             className="resize-y group absolute inset-1 flex flex-col overflow-y-auto rounded-lg bg-blue-50 p-2 text-xs leading-5 hover:bg-blue-100"
             draggable={true} >
           <p className="order-1 font-semibold text-blue-700">Breakfast</p>
           <p className="text-blue-500 group-hover:text-blue-700">
           <time dateTime="2022-01-14T06:00">6:00 AM</time>
          </p>
       </a>
   </li>
</ol>
```

Update: I applied `<li className="relative mt-px flex sm:col-start-3" style={{ gridRow: ' 72/span 18 minmax(min-content, max-content)' }} >` but cannot now initially seed the event to "start" at the right time on the calendar - now every event starts at midnight...

**Comment by liamgsmith** [comment]
Hey, I'm trying to unpick this css at the moment to make it more dynamic as well. Not sure if this will help/you probably noticed this!

What I've determined through just manual testing is that the gridRow is both starting location and also duration; 74 is the starting row and span 18 is the duration. 
I'm guessing this is set in the repeat(288, minmax(0,1fr) bit but I haven't had a good look to decipher that bit; I'm guesing it's the 1fr bit.

An hour = span 12, half hr 6, 45mins is 9
Midnight is gridRow 2, 1am is 14, 2am is 26; so I'm thinking it's start at 2, increment 12 per hour using 24hr time; eg 11pm is 23 * 12+2, 4pm is 16 * 12+2
